plumbing/api.go

```diff
@@ -162,6 +165,21 @@ func (api *API) DealPut(storageDeal *storagedeal.Deal) error {
 	return api.storagedeals.Put(storageDeal)
 }
 
+// OutboxQueues lists addresses with non-empty outbox queues
```

Comment:
maybe note the nondeterminism of the order here to save someone a little heartache down the line